### PR TITLE
(WIP)Docs migration

### PR DESCRIPTION
**Pull request by adreyer**
This is for reference only while we're working on the docs_migration branch.

### CODE REVIEW

File: pre-docs/bolt_new_features.md [hunk 1]
```diff
@@ -0,0 +1,417 @@
+# New features
```
Comment (adreyer) [inline pre-docs/bolt_new_features.md]: this file should be deleted right?

File: pre-docs/bolt.ditamap [hunk 2]
```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE map PUBLIC "-//OASIS//DTD DITA Map//EN" "map.dtd">
+<map id="boltmap" title="Bolt">
+    <topicref href="bolt.md" format="markdown"/>
+    <topicref href="bolt_release_notes.md" format="markdown"/>
```
Comment (adreyer) [inline pre-docs/bolt.ditamap]: It looks like this page was deleted? Does this build?

File: pre-docs/writing_plugins.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
+# Writing plugins
```
Comment (adreyer) [inline pre-docs/writing_plugins.md]: This duplicates "plugins.md"

File: pre-docs/bolt_new_features.md (hunk 1 above)
Comment (cwcowellshah) [reply pre-docs/bolt_new_features.md]: Yup. Was keeping it around for safety, but I've deleted it now in this branch.

File: pre-docs/writing_plugins.md (hunk 3 above)
Comment (cwcowellshah) [reply pre-docs/writing_plugins.md]: Good catch. I've removed it from this branch.

File: pre-docs/bolt.ditamap (hunk 2 above)
Comment (cwcowellshah) [reply pre-docs/bolt.ditamap]: I've re-added the page and reworked it a little to reflect our current organization, including the use of `CHANGELOG.md`.

**Pull request merged**